tests/cpu_bugs/xen_domu_mitigation_test.pm

```diff
@@ -703,69 +703,94 @@ sub install_domu {
               . " --serial pty", timeout => 5400);
     }
     # Check if the DomU is up
-    script_run("virsh start \"${domu_name}\"");
+    my $vmstatus = script_output("virsh start \"${domu_name}\"");
+    if ($vmstatus =~ /started/) {
+        print "DomU started successfully.";
+    } else {
+        print "DomU started fail,Please check the status on openqa snapshot and logs.";
+        print $vmstatus;
+    }
 }
 
 
 sub get_domu_ip {
     return script_output("./get_guest_ip.sh \"${domu_name}\"", timeout => 1800);
 }
 
+sub verifyIpaddress {
+    if ($_[0] =~ /\d+\.\d+\.\d+\.\d+/) {
+        print "Successfully and the ip is $_[0]\n";
+        return 1;
+    } else {
+        print $_[0] . "\n";
+        return 0;
+    }
+}
+
 sub exec_testcases {
     my $self = @_;
-
+    print "Begin installing domu\n";
     install_domu();
     $domu_ip_addr = get_domu_ip();
-
+    my $retip = verifyIpaddress($domu_ip_addr);
```

Comment:
Please refer to get_domu_ip function, the function already fetch ip address from guest inside, if you want to make more accurate verification, please add the checkpoint to get_domu_ip function.

Reply:
Refer to:http://10.67.129.4/tests/59033#step/xen_domu_mitigation_test/44. I have seen the get_domu_ip function but it will return others when couldn't get the ip. So I think openqa should add other function to check the ip.